src/config.ts

```diff
@@ -386,6 +386,19 @@ const BaseConfigSchema = z.object({
     .args(z.custom<Page>(), StoryLikeSchema)
     .returns(z.promise(z.void()))
     .optional(),
+
+  /**
+   * Perform actions after screenshot was shot
+   */
+  afterScreenshot: z
+    .function()
+    .args(
+      z.object({
+        shotPath: z.string(),
+      }),
```

Comment:
To keep it more flexible in the future, I'd suggest changing that to:
```
.args(z.custom<Page>(), StoryLikeSchema)
```

and extend the `StoryLikeSchema` with `filePathBaseline`, `filePathCurrent`, and `filePathDifference` (all optional props).

Reply:
ready